REDIS_PRODUCTION.md

```diff
@@ -0,0 +1,33 @@
+# Redis Memory Configuration for Production
```

Comment:
I like the idea of this document overall, but maybe we can make the top level title and filename into a more general purpose `PRODUCTION_GUIDE.md`/ "guide for runing 8Knotin production" or something and make this a second level subheading or something